template/base/.env-example

```diff
@@ -4,5 +4,9 @@
 DATABASE_URL=
 
 # Next Auth
+NEXTAUTH_SECRET=
+NEXTAUTH_URL=
+
+# Github
```

Comment:
The GITHUB_CLIENT and SECRET are still NextAuth env-vars

Reply:
I moved them out because I thought they are more generic (like GH_TOKEN for workflows), but they belong to the GithubProvider, so either section makes sense to me. Should I move them back?

Reply:
Its not a big deal and I dont know whats more confusing: having a Github variable for Auth or having it separate and left wonder what it does.

Reply:
Case in point, it confused me too. I renamed it, how about now?